src/tokens/functional/motion/patterns.json5

```diff
@@ -0,0 +1,26 @@
+/**
+ * Motion patterns
+ * @description Used to create animated components
+ */
+{
+  motion: {
+    spinner: {
+      rotationDuration: {
```

Comment:
I am not sure, but WDYT think about a more general approach like `spinner.speed` and `skeletonLoader.speed`? 

CC: @mperrotti @langermank

Reply:
I feel like `duration` is different from `speed` in animation

Reply:
Than maybe using just `duration` instead of `rotationDuration` and `shimmerDuration`?

Reply:
I think `duration` could work for this case. When we start to add tokens for more complex animations, we might need to specify _what_ the duration is for.

For example: If I have an animation that scales in (`transform: scale()`) at a different rate than it fades in (`opacity`), we'll need to differentiate the two durations.

Reply:
Yes, true. But we could also think about nested naming? E.g. maybe this is part of the "scale"?

`duration.scale` and `duration.opacity`? 

Do you think we need to figure this out now, or should we go with `duration` and see if we ever run into issues?

Reply:
So `motion-spinner-rotationDuration-default` could be `motion-spinner-duration-rotation`

with future tokens like..

`motion-spinner-duration-scale` or `motion-spinner-duration-opacity`

I like this.